contracts/utils.gno

```diff
@@ -0,0 +1,73 @@
+package gnoswap_optimizer
+
+import (
+	"fmt"
+	"strconv"
+
+	u256 "gno.land/p/gnoswap/uint256"
```

Comment:
I would stick to uint256 for the pkg name (u256 sounds rust-y :). If `uint256.Uint` bothers you, maybe you can alias it with a type in your pacakge.